src/bin/contribute.rs

```diff
@@ -985,6 +987,32 @@ impl Contribute {
             self.notify_contribution(&chunk_id, serde_json::to_value(signed_data)?)
                 .await?;
 
+            if chunk_id == "0" {
+                // load data from attestation
+                match fs::read_to_string("plumo.attestation") {
+                    Ok(line) => {
+                        let arr: Vec<&str> = line.split(" ").collect();
```

Comment:
What happens if the name has spaces?
Maybe we should create a struct for the attestation, that is generated anyway in this binary, and use that to feed both the file and this method, rather than reading the file and parsing it?